parser.py

```diff
@@ -123,13 +129,27 @@ def show(self, data):
 def main()-> int:
     parser = argparse.ArgumentParser(description='Composer Symphony text parser')
     parser.add_argument('-i','--infile', dest="infile", action="store", help=' input file we read the symphony text from.  full path please', required=True)
+    parser.add_argument('-u', '--url', action="store_true", help="specifies that the input file path is actually the url to a shared, public symphony on composer.trade")
     parser.add_argument('-o','--outfile', dest="outfile", action="store", default="OUTFILE", help=' output file to save the parsed text to.  if not given, will use stdout', required=False)
     parser.add_argument('-m','--mode', dest="mode", action="store", default="human", help=' output parsing mode to use.  if none given, will parse for "human readable output".  modes are: quantconnect, vectorbt, tradingview, thinkscript', required=False)
     args = vars(parser.parse_args())
 
+    if args['url'] == True:
+        #todo move to request this from the composer site instead of hardcoding
+        composerConfig = {
+                "projectId" : "leverheads-278521",
+                "databaseName" : "(default)"
+            }
+        m = re.search('\/symphony\/([^\/]+)', args["infile"])
+        symphId = m.groups(1)[0]
+        symphReq = requests.get(f'https://firestore.googleapis.com/v1/projects/{composerConfig["projectId"]}/databases/{composerConfig["databaseName"]}/documents/symphony/{symphId}')
+        resp = json.loads(symphReq.text)
```

Comment:
`response.json()` might do what you want
Might help to do `response.raise_for_status()` if we get an unexpected HTTP code